gallery/visualtests/nxAlert.js

```diff
@@ -17,24 +17,36 @@ describe('NxAlert', function() {
       noCloseSelector = '#nx-alert-no-close-example .nx-alert',
       withLinkSelector = '#nx-alert-with-link-example .nx-alert';
 
+  const runTests = (selector) => {
+    it('looks right', simpleTest(selector));
+    it('looks right with the close button hovered', hoverTest(selector, `${selector} .nx-btn--close`));
+    it('looks right with the close button focused', focusTest(selector, `${selector} .nx-btn--close`));
+    it('looks right with the close button clicked', clickTest(selector, `${selector} .nx-btn--close`));
+    it('looks right with the close button focused and hovered',
+        focusAndHoverTest(selector, `${selector} .nx-btn--close`));
+  };
+
   describe('Custom NxAlert', function() {
-    it('looks right', simpleTest(simpleSelector));
+    runTests(simpleSelector);
   });
 
   describe('NxSuccessAlert', function() {
-    it('looks right', simpleTest(successSelector));
+    runTests(successSelector);
   });
 
   describe('NxErrorAlert', function() {
-    it('looks right', simpleTest(errorSelector));
-    it('looks right with the close button hovered', hoverTest(errorSelector, `${errorSelector} .nx-btn--close`));
-    it('looks right with the close button focused', focusTest(errorSelector, `${errorSelector} .nx-btn--close`));
-    it('looks right with the close button clicked', clickTest(errorSelector, `${errorSelector} .nx-btn--close`));
-    it('looks right with the close button focused and hovered',
-        focusAndHoverTest(errorSelector, `${errorSelector} .nx-btn--close`));
+    runTests(errorSelector);
+  });
+
+  describe('NxInfoAlert', function() {
+    runTests(infoSelector);
   });
 
-  describe('NxLinkInAlert', function() {
+  describe('NxWarningAlert', function() {
+    runTests(warningSelector);
+  });
+
+  describe('with a link', function() {
```

Comment:
Since these are renamed, the corresponding image files should be moved to match the new name, avoiding the need to add even more image files in this PR

Reply:
> Since these are renamed, the corresponding image files should be moved to match the new name, avoiding the need to add even more image files in this PR

Deleted the older files in 65de78a53074deff62cbf6e0002568664da2363f
The PR should now just show renamed files.